src/core/layout/footer/styles.jsx

```diff
@@ -1,9 +1,9 @@
 import { makeStyles } from '@material-ui/core'
```

Comment:
Shouldn't this file have the `.js` extension, since we're not using any jsx within it?

Reply:
Done